irohad/ordering/ordering_types.hpp

```diff
@@ -12,13 +12,30 @@
 
 namespace iroha::ordering {
 
-  using BloomFilter256 = shared_model::crypto::BloomFilter<
-      shared_model::crypto::Hash,
-      256,
-      shared_model::crypto::Iroha2BloomHasher64<0, 32>,
-      shared_model::crypto::Iroha2BloomHasher64<1, 32>,
-      shared_model::crypto::Iroha2BloomHasher64<2, 32>,
-      shared_model::crypto::Iroha2BloomHasher64<3, 32>>;
+#define USE_BLOOM_FILTER 0
+
+  static constexpr uint64_t kBloomFilterOrder = 32 * 1024ull;
+  using BloomFilter256 =
+      shared_model::
+          crypto::BloomFilter<shared_model::crypto::Hash,
+                              kBloomFilterOrder,
+                              shared_model::crypto::
+                                  Iroha2BloomHasher64<0, kBloomFilterOrder / 8>,
```

Comment:
May be add constant for `kBloomFilterOrder / 8`, for expamle `kBloomFilterSize`. I think code would be more clear.